wrc/codegen/cghtml.py

```diff
@@ -76,8 +76,10 @@ def link2html(text):
 def simple_md2html(text, urls):
     ''' Convert a text from md to html '''
     retval = special_links_replace(text, urls)
-    # Create a br for every 4 spaces
-    retval = re.sub(r'[ ]{4}\n', r'<br />\n', retval)
+    # Create a par break for double newlines
+    retval = re.sub(r'\n\n', r'</p><p>', retval)
```

Comment:
Given the way Regulations are compiled right now we can ensure this, because `md2html` is used only in the following contexts:
- Titles' text
- Regulations/Guidelines text
- Other sections' text

For the first two, having '\n\n' in the text is simply impossible given the current token description. For the last context the function visiting a text is `visitunicode` and encapsulate it in `<p></p>`:
```python
    def visitunicode(self, u):
        if len(u) > 0:
            self.codegen += "<p>" + self.md2html(u) + "</p>\n"
        return True
```